web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -53,16 +56,21 @@ export default {
       return !(this.name && this.description);
     },
   },
+  mounted() {
+    if (!loggedIn()) {
+      this.$router.push({name: 'home'});
+    }
+  },
   methods: {
     async register_dandiset() {
       const { name, description } = this;
-      const { status, data } = await this.girderRest.post('dandi', null, { params: { name, description } });
+      const { status, data } = await girderRest.post('dandi', null, { params: { name, description } });
 
       if (status === 200) {
         this.name = '';
```

Comment:
You can probably remove these, since the component should be destroyed when navigating to another view.

They are a holdover from when this form was embedded in a global dialog.